commands.md

```diff
@@ -1,28 +1,28 @@
 # Manual without Docker
 
-npm run devServer -- --ip localhost --port 8440 --id 0 --targetIp localhost --targetPort 8440 --targetId 0
-npm run devClient -- crawl --ip localhost --port 8440 --webPort 1337
+npm run devServer -- --host localhost --port 8440 --knownHost localhost --knownPort 8440
+npm run devClient -- crawl --host localhost --port 8440 --webPort 1337
 
-npm run devServer -- --ip localhost --port 8441 --id 6 --targetIp localhost --targetPort 8440 --targetId 0
-npm run devServer -- --ip localhost --port 8442 --id 1 --targetIp localhost --targetPort 8440 --targetId 0
-npm run devServer -- --ip localhost --port 8443 --id 2 --targetIp localhost --targetPort 8440 --targetId 0
-npm run devServer -- --ip localhost --port 8445 --id 3 --targetIp localhost --targetPort 8440 --targetId 0
-npm run devServer -- --ip localhost --port 8444 --id 7 --targetIp localhost --targetPort 8440 --targetId 0
+npm run devServer -- --host localhost --port 8441 --knownHost localhost --knownPort 8440
+npm run devServer -- --host localhost --port 8444 --knownHost localhost --knownPort 8440
+npm run devServer -- --host localhost --port 8446 --knownHost localhost --knownPort 8440
+npm run devServer -- --host localhost --port 8448 --knownHost localhost --knownPort 8440
+npm run devServer -- --host localhost --port 8450 --knownHost localhost --knownPort 8440
 
 <!-- Alvaro: npm run devClient worked for me, but it does not ends so I can't run the following command -->
 
-node client insert --ip localhost --port 8440 --id 2
-node client insert --ip localhost --port 8440 --id 5 --displayName "Alvaro is cool" --reputation 99 --aboutMe "I'm so cool I need no description"
-node client lookup --ip localhost --port 8440 --id 2
-node client remove --ip localhost --port 8440 --id 2
+node client insert --host localhost --port 8440
```

Comment:
`id` has been a bit overloaded for us, but these values are actually the `id` of the user that we're inserting, updating, looking up, removing, etc. So these actions fail currently. You need to put the id arguments back for 14-17

Reply:
reverted as requested

Reply:
reverted as requested